browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -236,13 +236,15 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 			$('#applystyle').val(state).trigger('change');
 		}
 		else if (commandName === '.uno:ModifiedStatus') {
-			if (e.state === 'true') {
-				$('#Save1').addClass('savemodified');
-				$('#Save').addClass('savemodified');
-			}
-			else {
-				$('#Save1').removeClass('savemodified');
-				$('#Save').removeClass('savemodified');
+			if (document.getElementById('save')) {
+				if (state === 'true') {
+					document.getElementById('save').classList.add('savemodified');
+					document.getElementById('file-save').classList.add('savemodified');
+				}
+				else {
+					document.getElementById('save').classList.remove('savemodified');
+					document.getElementById('file-save').classList.remove('savemodified');
```

Comment:
I thought there is no need to check if file-save element exists since if we are in presence of notebookbar then the File tab should always exist, right? Or should we anyhow check it exists .
In the pass we didn't check at all.